docs/_providers/internetbs.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Internet.bs
+title: Internet.bs Provider
+layout: default
+jsId: INTERNETBS
+---
+# Internet.bs Provider
+
+Internet.bs has API to mange your DNS zones. But in current version of provider, it's not implemented. You can use Internet.bs only as registrar for now.
+
+
+## Configuration
+In your credentials file, you must provide your API key and account password 
+
+{% highlight json %}
+{
+  "internetbs": {
+    "api-key": "your-api-key",
+    "password": "account-password"
+  }
+}
+{% endhighlight %}
+
+## Metadata
+This provider does not recognize any special metadata fields unique to Internet.bs.
+
+## Usage
+Example Javascript:
+
+{% highlight js %}
+var REG_INERNETBS = NewRegistrar('internetbs', 'INTERNETBS');
+var GCLOUD = NewDnsProvider("gcloud", "GCLOUD"); // Any provider
```

Comment:
Suggestion: use the "NONE" provider instead.

Reply:
ups, thanks :)

Reply:
That would be great, agree.  But...  There is a  "NONE" registrar. But I could not find any mentions of "NONE" **provider**.

```js
var DNS_NONE = NewDnsProvider("nonee", "NONE");
```
gives me 
> DSP type NONE not declared

could you help with example ?

Reply:
I was wrong.  There is no NONE provider for Registratars, just DNS providers.   I'll have to consider adding one!

In the meanwhile, this example is sufficient.